toolkit/tools/imagecustomizerschemacli/go.mod

```diff
@@ -0,0 +1,26 @@
+module jsonschemacli
```

Comment:
Please don't add a new `go.mod` file. It doesn't cause any harm to include this as part of the existing `go.mod` file.

Reply:
I thought to make this independent of the existing tools, but no strong opinion.

Reply:
Some of the Go tooling doesn't like multiple `go.mod` files.